sigs.yaml

```diff
@@ -975,6 +975,67 @@ sigs:
     - https://raw.githubusercontent.com/kubernetes/kubernetes-anywhere/master/OWNERS
   - name: kubespray
     contact:
+      slack: sig-docs
```

Comment:
@mittalyashu these changes don't look correct (they are under kubespray).

Can you fix this and squash the commits?

Reply:
> @mittalyashu these changes don't look correct (they are under kubespray).

That's weird, let me fix that.

> Can you fix this and squash the commits?

After the final review, I will squash the commits.

Reply:
I have just checked the history of the `sigs.yaml` file and somethings fishy going on there 😅.